src/Compilers/CSharp/Portable/Symbols/Metadata/PE/PETypeParameterSymbol.cs

```diff
@@ -705,5 +707,20 @@ private NamedTypeSymbol GetDefaultBaseType()
         {
             get { return null; }
         }
+
+#nullable enable
+        internal override UseSiteInfo<AssemblySymbol> GetUseSiteInfo()
```

Comment:
>GetUseSiteInfo [](http://example.com/codeflow?start=54&length=14)

It looks like we are adding this error to declaring method/type by explicitly iterating through the type parameters anyway. I think there is no reason to override this method or cache information on this symbol.